src/lib/utils/index.ts

```diff
@@ -312,6 +312,9 @@ export const getTimeFrameBounds = (timeframe: HistoryTimeframe): { start: Dayjs;
 export const chainIdToFeeAssetId = (chainId: ChainId): AssetId | undefined =>
   getChainAdapterManager().get(chainId)?.getFeeAssetId()
 
+export const chainIdToChainDisplayName = (chainId: ChainId): AssetId | undefined =>
+  getChainAdapterManager().get(chainId)?.getDisplayName()
+
```

Comment:
We can use `assertGetChainAdapter` here instead and remove the undefined union

Reply:
sounds good!